src/v3/src/util/formUtils.ts

```diff
@@ -492,3 +525,144 @@ export const buildPhoneVerificationSubtitleElement = (
 
   return subtitleElement;
 };
+
+type RequestOptions = {
+  url: string;
+  timeout: number;
+  method: 'GET' | 'POST';
+  data?: string;
+};
+
+/**
+* Temporary request client can be removed if this functionality is added
+* to auth-js library.
+* @see https://oktainc.atlassian.net/browse/OKTA-561852
+* @returns Promise<Response>
+*/
+export const makeRequest = async ({
```

Comment:
Is this abortable fetch part of the requirement? or a good to have? maybe consider introduce a well tested package for it?

Reply:
Being able to abort or cancel after the timeout has elapsed is more of a generic requirement for the loopback server. 

This method was actually copied from `LoopbackProbe.tsx` as I didn't want to duplicate this.

It looks like the [ticket](https://oktainc.atlassian.net/browse/OKTA-561852) that was created to replace this was resolved as "Won't do". IIRC we didnt want to pull in an additional api client lib (at the risk of increasing the pkg size), and since cross-fetch is already used by Gen 2 we just used what was already there.